openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -824,6 +828,8 @@ The JWT MUST contain the following elements:
 
 The Credential Issuer MUST validate that the JWT used as a proof is actually signed by a key identified in the JOSE Header.
 
+If an `attestation` is provided and successfully validated by the Credential Issuer, it SHOULD return a Credential for each of the keys provided in the `attested_keys` claim of the attestation.
```

Comment:
Is there a requirement that the key doing the signing of the proof must be an attested key from the set of `attested_keys`?

Asking because the spec defines `proof` as:

https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0-14.html#section-7.2-2.3.1 
> Object providing a single proof of possession of the cryptographic key material to which the issued Credential instance will be bound to...

definition of`proofs` then builds on top of that definition.